transformers/component_tests/queries/all_flips_query_test.go

```diff
@@ -151,52 +208,50 @@ var _ = Describe("All flips view", func() {
 		})
 		Expect(setupErr).NotTo(HaveOccurred())
 
-		bidIdTwo := bidIdOne + 1
-		ilkTwo := test_helpers.AnotherFakeIlk
-		anotherFlipAddress := "flip2"
+		anotherFlipAddress := fakes.AnotherFakeAddress.Hex()
 
 		headerTwo := createHeader(blockOne+1, timestampOne+1, headerRepo)
 
-		flipStorageValuesTwo := test_helpers.GetFlipStorageValues(2, ilkTwo.Hex, bidIdTwo)
-		test_helpers.CreateFlip(db, headerTwo, flipStorageValuesTwo, test_helpers.GetFlipMetadatas(strconv.Itoa(bidIdTwo)), anotherFlipAddress)
+		flipStorageValuesTwo := test_helpers.GetFlipStorageValues(2, ilkOne.Hex, fakeBidIdTwo)
+		test_helpers.CreateFlip(db, headerTwo, flipStorageValuesTwo, test_helpers.GetFlipMetadatas(strconv.Itoa(fakeBidIdTwo)), anotherFlipAddress)
 
 		// insert a new bid associated with a different flip contract address
 		ilkIdTwo, urnIdTwo, setupErr := test_helpers.SetUpFlipBidContext(test_helpers.FlipBidContextInput{
 			DealCreationInput: test_helpers.DealCreationInput{
 				DB:              db,
-				BidId:           bidIdTwo,
+				BidId:           fakeBidIdTwo,
```

Comment:
As a matter of fact, I wonder if these two records should have the same bid ID, so that they only differ on the address 🤔

Reply:
Good call! I ended up adding a new test for this, which may be overkill, but it seems like it'd be nice to test the scenario where the bids have the same id, and when they have different ids.